src/arch/arm/object/vcpu.c

```diff
@@ -447,6 +421,7 @@ void vcpu_init(vcpu_t *vcpu)
 #endif
     /* GICH VCPU interface control */
     vcpu->vgic.hcr = VGIC_HCR_EN;
+    vcpu->vgic.vmcr = 0xff000002; // TODO @yanyan wtf
```

Comment:
set default values for virtual EL1 PMR and enable group 1 virtual interrupts.

Reply:
Ok. Updated. Does this need to be only for the gicv3 or for both?